src/Kentico.Xperience.Lucene/Admin/Client/package.json

```
@@ -12,8 +12,8 @@
     "node": ">=18.12.0 <=19"
   },
   "dependencies": {
-    "@kentico/xperience-admin-base": "26.4.0",
-    "@kentico/xperience-admin-components": "26.4.0",
+    "@kentico/xperience-admin-base": "^28.0.2",
+    "@kentico/xperience-admin-components": "^28.0.2",
```

```suggestion
    "@kentico/xperience-admin-base": "28.0.2",
    "@kentico/xperience-admin-components": "28.0.2",
```

These should be explicit, not floating versions.

---

I resolved this